addons/skin.estuary/xml/MusicOSD.xml

```diff
@@ -76,7 +76,7 @@
 						<param name="texture" value="osd/fullscreen/buttons/next.png"/>
 					</include>
 					<onclick>PlayerControl(Next)</onclick>
-					<visible>MusicPlayer.HasNext + !MusicPlayer.Content(livetv)</visible>
+					<visible>Player.SeekEnabled | MusicPlayer.HasNext</visible>
```

Comment:
during music playback, the next button will now be visible even if there is no next song to play.

Reply:
perhaps somthing like this would work?
`<visible>[Player.SeekEnabled + MusicPlayer.Content(livetv)] | MusicPlayer.HasNext</visible>`

Reply:
Right. I will fix that.